the-list-be/server.js

```diff
@@ -1,19 +1,39 @@
 import Koa from 'koa';
 import Router from '@koa/router';
+import { MongoClient } from 'mongodb';
+import koaBody from 'koa-body';
 
 const app = new Koa();
 const router = new Router();
 
+const uri = 'mongodb://localhost:27017';
+const client = new MongoClient(uri);
+const db = client.db('database');
+
+const students = db.collection('students');
+
+router.post('/student', koaBody(), async ctx => {
+  const insertResult = await students.insertOne(ctx.request.body);
+  ctx.body = `Insert documents => ${insertResult}, Request body => ${JSON.stringify(ctx.request.body)}`;
```

Comment:
Здесь имеет больше смысла просто отправить вставленный документ обратно клиенту.
```suggestion
  ctx.body = insertResult;
```

Reply:
@OstaptsovDanil, ты уверен, что хочешь это оставить в таком виде?